addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/internal/NestUpdateRequest.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.nest.internal;
+
+import java.util.Map;
+
+/**
+ * Contains the data needed to do an update request back to nest.
+ *
+ * @author David Bennett - Initial Contribution
+ */
+public class NestUpdateRequest {
+    public String getUpdateUrl() {
+        return updateUrl;
+    }
+
+    public void setUpdateUrl(String updateUrl) {
+        this.updateUrl = updateUrl;
+    }
+
+    public Map<String, Object> getValues() {
+        return values;
+    }
+
+    public void addValue(String key, Object value) {
+        values.put(key, value);
+    }
+
+    private String updateUrl;
```

Comment:
Please keep the order more or less like the following:

1. logger
2. static fields
3. normal fields
4. constructor
5. other methods, grouped with usage or some other sensible way ordered